pytest_splunk_addon/splunk.py

```diff
@@ -101,6 +101,13 @@ def pytest_addoption(parser):
             " of 8.0.0. "
         ),
     )
+    group.addoption(
+        "--dm-path",
+        action="store",
+        dest="dm_path",
+        default="pytest-splunk-addon\pytest_splunk_addon\standard_lib\datamodels",
+        help=("Path to the datamodels directory"),
```

Comment:
Add the detail that 
- relative or absoute path can be provided. 
- Json files are expected in the directory.
- Where can a user find the schema for those json.

Reply:
Done.